### PR TITLE
feat: updates related to new keycloak version

### PR DESCRIPTION
**Pull request by Ani1357**
Renamed the KEYCLOAK_ADDRESS to KC_HOSTNAME as per new version of keycloak.

### CODE REVIEW

File: .env.sample [hunk 1]
```diff
@@ -20,7 +20,7 @@ IDP_CLIENT_SECRET=''
 IDP_GROUP_OTOMI_ADMIN='e69ded30-0882-4490-8e0f-2e67625a0693'
 IDP_GROUP_TEAM_ADMIN='3c63814c-59df-46c3-9a69-d9e1c3611097'
 IDP_GROUP_MAPPINGS_TEAMS='{"team-otomi":"28010af7-9535-4265-8689-50f51f8f2c87"}'
-KEYCLOAK_ADDRESS='https://keycloak.dev.gke.otomi.cloud'
+KC_HOSTNAME='https://keycloak.dev.gke.otomi.cloud'
```
Comment (j-zimnowoda) [inline .env.sample:23]: if it is a hostname, then why in the example there is an URL ?

**Review by j-zimnowoda — CHANGES_REQUESTED** [review]

File: .env.sample (hunk 1 above)
Comment (Ani1357) [reply .env.sample]: updated it to [KC_HOSTNAME_URL](https://www.keycloak.org/server/all-config?q=hostname#category-hostname)

**Comment by ElderMatt** [comment]
Previously the SSO tokens were set to 8hours but this is not the case anymore. 
image
image

**Comment by ElderMatt** [comment]
At the moment Teams/groups are not automatically added in keycloak. This happens in keycloak.ts, search for 'create groups' then you will find it.

**Comment by Ani1357** [comment]
@ElderMatt you are checking the master realm in keycloak instead of otomi. Tested again and teams are being added as expected and also the correct TTL for the SSO is there.

**Review by j-zimnowoda — APPROVED** [review]

**Pull request merged**